test/parallel/test-tls-multi-key.js

```diff
@@ -21,21 +21,21 @@
 
 'use strict';
 const common = require('../common');
+const fixtures = require('../common/fixtures');
```

Comment:
Nit: would you mind moving this after the `common.hasCrypto` check?

Reply:
Not at all! Done!